game/helipad.py

```diff
@@ -0,0 +1,23 @@
+from typing import Optional
+
+from dcs import Point
+from dcs.unitgroup import StaticGroup
+
+from game.point_with_heading import PointWithHeading
+from game.utils import Heading
+
+
+class Helipad(PointWithHeading):
+    def __init__(self) -> None:
+        super(Helipad, self).__init__()
+        self.heading = Heading.from_degrees(0)
+        self.occupied = False
+        self.static_unit: Optional[StaticGroup] = None
+
+    @staticmethod
+    def from_point(point: Point, heading: Heading) -> "Helipad":
```

Comment:
Minor optional cleanup: `from __future__ import annotations` at the top and then you can drop the quotes on the type

(once Python 3.10 releases in October that's finally available by default)

Reply:
Oh so that's why 😮, I thought i was going crazy with this one, found no other way than to quote it while it was working everywhere else.
Thanks !

Reply:
Resolved in 9ea1edf9dbc8b21c7204ca6a71eab847ddc63522